docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,231 @@
+If enabled, by setting ``wifi5.outdoor_chanlist``, a number of devices that are commonly installed outdoors will have outdoor mode automatically enabled during their initial setup, specifically:
+
+* Ubiquiti
+
+    - Bullet M
+    - Litebeam M5
+    - Nanostation M5
+    - Nanostation M5 Loco
+    - Rocket M5
+    - Rocket M5 TI
+    - Unifi AC Mesh
+    - Unifi AC Mesh Pro
+    - Unifi Outdoor
+
+* TP-Link
+
+    - CPE510
+    - WBS510
+
+See the :ref:`wifi5 <user-site-wifi5>` section for the *site.conf* configuration of this feature.                  
+
+
+Hoodselector: Geolocation Mode
+==============================
+
+Hoodselector is a new concept for continuous reevaluation of the nodes domain. As a first step we support its geolocation feature.
+
+See the :doc:`../package/gluon-hoodselector` documentation for details.
+
+
+Device Deprecation
+==================
+
+The ar71xx-tiny and several devices in the ramips-rt305x target have been marked as deprecated. The `GLUON_DEPRECATED` flag was introduced to offer downstream communities the choice on how to deal with the ending support for those devices.
+
+See the :ref:`Build configuration <user-site-build-configuration>` section for details.
+
+
+Debug Build-Flag
+================
+
+Setting ``GLUON_DEBUG=1`` will provide firmware images including debugging symbols usable with GDB or similar tools. Requires a device or target with at least 16 MB of flash space, e.g. `x86-64`. Unset by default.
+
+x86 images support firstboot
+============================
+
+x86 images are now using squashfs over ext4 and can now have their configuration reset by using ``firstboot``.
+
+
+Bugfixes
+********
+
+* Fixes passwordless SSH access when gluon-authorized-keys was used without gluon-setup-mode. (`#1777 <https://github.com/freifunk-gluon/gluon/issues/1777>`_)
+
+- Fixes cross-domain leakage of respondd data by not joining the link-local multicast group on br-client. Nodes now only listen on ``[ff05::2]:1001`` for respondd queries. This may require an update to your respondd querying tooling. (`#1701 <https://github.com/freifunk-gluon/gluon/issues/1701>`_)
+
+
+Site changes
+************
+
+When updating a site configuration from Gluon 2018.2.x, the following changes
+must be made:
+
+site.mk
+=======
+
+- We now require the ``GLUON_DEPRECATED`` variable to be set to decide how to handle the image generation for deprecated devices. (`#1745 <https://github.com/freifunk-gluon/gluon/pull/1745>`_)
+
+site.conf
+=========
+
+- The options ``wifi*.basic_rates`` and ``wifi*.supported_rates`` have been removed, as the legacy 802.11b rates are now disabled by default. (`#1716 <https://github.com/freifunk-gluon/gluon/pull/1716>`_)
+
+
+Internals
+*********
+
+BATMAN-adv multicast optimizations
```

Comment:
like with the above "Batman 14" i'm also not sure here, if BATMAN-adv is correct naming/spelling

Reply:
Going with `batman-adv` in both cases, the name of the kmod.